client/index.html

```diff
@@ -1,31 +1,37 @@
 <!DOCTYPE html>
 <html>
+
   <head>
     <meta name="charset" content="utf-8">
     <title>Q</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="icon" type="image/png" href="/favicon.png">
-    <base href="/">
+    <base href="https://localhost:9000/proxy/q-editor/">
```

Comment:
The regex here here should still work according to the regex in the link below, but maybe we could change to the following. https://github.com/livingdocsIO/q-editor/blob/master/routes/index.js#L15
`/^\s*<base.*$/m`

Reply:
What do you think about `^\s*<base.*>` to also not fail if anyone adds things on the same line as the base element?

Reply:
Yes, that's probably safe here as nobody puts a `>` into a url 😁

Reply:
Nope, that second one won't work. regexes 🙄 
![image](https://user-images.githubusercontent.com/431376/70792966-9e866b80-1d9a-11ea-8174-8bb401142099.png)

Reply:
👍

Reply:
5a8469fad8d7328d6a0a3ac6ef1b71c6e5ffc4d4